validation/validation_template.env

```diff
@@ -0,0 +1,24 @@
+# LOG_LEVEL = "INFO"
```

Comment:
can we just pull env variables, particularly for the DB details, from the main env file? Otherwise I suspect that we will have to do double work in making sure that all the DB details match across multiple env files.

Reply:
Yeah the database env file for configs is in the front-end repo sadly. Given this is open source and the other one is not, maybe manual will need to be done.

In the algorithm repo, I don't think we have a template file anywhere